posthog/api/cohort.py

```diff
@@ -205,6 +213,30 @@ class LegacyCohortViewSet(CohortViewSet):
     legacy_team_compatibility = True
 
 
+def will_create_loops(cohort: Cohort, new_properties: PropertyGroup) -> bool:
+    # Loops can only be formed when trying to update a Cohort, not when creating one
```

Comment:
unless there was an invalid ID existing earlier, and that ID is miraculously this new cohort's ID. hmmm.